packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -159,39 +162,26 @@ export function getMomentTemplateBanner(
                     </div>
                 </Container>
 
-                <Hide above="tablet">
-                    <div ref={mobileReminderRef}>
-                        {content.mobileContent.secondaryCta?.type ===
-                            SecondaryCtaType.ContributionsReminder &&
-                            isReminderActive && (
-                                <MomentTemplateBannerReminder
-                                    reminderCta={content.mobileContent.secondaryCta}
-                                    trackReminderSetClick={reminderTracking.onReminderSetClick}
-                                    setReminderCtaSettings={templateSettings.setReminderCtaSettings}
-                                />
-                            )}
-                    </div>
-                </Hide>
-
-                <Hide below="tablet">
-                    {content.mainContent.secondaryCta?.type ===
-                        SecondaryCtaType.ContributionsReminder &&
-                        isReminderActive && (
-                            <MomentTemplateBannerReminder
-                                reminderCta={content.mainContent.secondaryCta}
-                                trackReminderSetClick={reminderTracking.onReminderSetClick}
-                                setReminderCtaSettings={templateSettings.setReminderCtaSettings}
-                            />
-                        )}
-                </Hide>
+                {content[mainOrMobileContent].secondaryCta?.type ===
+                    SecondaryCtaType.ContributionsReminder &&
+                    isReminderActive && (
+                        <MomentTemplateBannerReminder
+                            reminderCta={
+                                content[mainOrMobileContent]
+                                    .secondaryCta as BannerEnrichedReminderCta
+                            }
+                            trackReminderSetClick={reminderTracking.onReminderSetClick}
+                            setReminderCtaSettings={templateSettings.setReminderCtaSettings}
+                            mobileReminderRef={mobileReminderRef}
```

Comment:
Presume its better to pass this reminder HTMLDivElement into the component rather than  put it above the MomentTemplateBannerReminder component call on line 168 above (as was previously?)?

Reply:
This is a really good catch! The reminder ref is only for mobile, previously there were the 2 separate sections for mobile and desktop. The ref is passed to the component to prevent the need to do that, but there is one last step missing here I believe. We want to do a conditional check to pass the mobileReminderRef only when it's on mobile. Will update now!